cmd/xmidt-agent/config.go

```diff
@@ -296,6 +302,9 @@ var defaultConfig = Config{
 			MaxInterval: 341*time.Second + 333*time.Millisecond,
 		},
 	},
+	Pubsub: Pubsub{
+		PublishTimeout: 200 * time.Millisecond,
+	},
 	MockTr181: MockTr181{
 		FilePath: "./mock_tr181.json",
 		Enabled:  true,
```

Comment:
I'm assuming we don't want MockTr181 to run by default
```suggestion
		Enabled:  false,
```